modules/elasticache-redis/variables.tf

```
@@ -75,6 +75,12 @@ variable "redis_clusters" {
   description = "Redis cluster configuration"
 }
 
+variable "allow_ingress_from_this_vpc" {
+  type        = bool
+  default     = false
```

```suggestion
  default     = true
```

We should not change default behavior except for:
- Major upgrades, with big warnings about breaking changes
- Previous default behavior no longer works or is being exploited by malicious users.

---

fair enough. Updated